scripts/ci/format-security-comment.py

```
@@ -0,0 +1,186 @@
+    lines = [
+        "### ❌ Scanner Error",
+        "",
+        "osv-scanner encountered an error during scanning. "
+        "Review the CI logs for details.",
+        "",
+    ]
+
+    raw = Path(raw_path)
+    if raw.exists():
+        content = raw.read_text(errors="replace")[:500]
+        content = content.replace("```", "ʼʼʼ")
```

Ruff (0.15.11)

[warning] String contains ambiguous `ʼ` (MODIFIER LETTER APOSTROPHE). Did you mean ``` (GRAVE ACCENT)?

(RUF001)